### ISSUE TITLE
Separate `Recording Settings` & `Timer Settings`

### ISSUE THREAD

**Issue by wedebe** [openatv/enigma2#2996]
**Is your feature request related to a problem? Please describe.**
I've been using OpenATV for years and still find it difficult to find certain settings.

**Describe the solution you'd like**
I'd like to see a new `Timer Settings` page, with certain items placed here and taken out of `Recording Settings`

This would also allow for accessing settings directly from the Record Timers Overview list

**Describe alternatives you've considered**
Better things I could be doing with my life in the time saved looking for settings

**Additional context**
![20230807123655](https://github.com/openatv/enigma2/assets/9741693/1b29cef5-f4e0-4bd6-8f85-bc24e1dd2db7)

![20230807124013-Recording1](https://github.com/openatv/enigma2/assets/9741693/cc8863aa-d7e3-481a-a9f2-4e5990c646f5)

![20230807124040-Recording2](https://github.com/openatv/enigma2/assets/9741693/fc907b97-b6e8-420e-a2ac-f4e0bd348834)

![20230807124013-Timers1](https://github.com/openatv/enigma2/assets/9741693/b9a095e4-860d-488a-80a4-baf7b0a5d870)

![20230807124040-Timers2](https://github.com/openatv/enigma2/assets/9741693/1156426a-bd6b-4d3a-aa08-665af0cba893)

**Comment by wedebe** [comment]
> Not good. IMHO. This will confuse the user.

I respectfully disagree. Finding `Zap` timer settings, for example, is unintuitive as they currently exist in `Recording Settings`, and recording margins only apply to timers. Can you expand on your reasoning for finding separation confusing?

**Comment by jbleyel** [comment]
> > Not good. IMHO. This will confuse the user.
> 
> I respectfully disagree. Finding `Zap` timer settings, for example, is unintuitive as they currently exist in `Recording Settings`, and recording margins only apply to timers. Can you expand on your reasoning for finding separation confusing?

You should ask the user and you will get the answer.

**Comment by atvcaptain** [comment]
https://github.com/openatv/enigma2/commit/85747deeb0540a0bcdae3be3cd1ffecc565b6c95

**Comment by IanSav** [comment]
I thing there are some issues with these new changes:
- On a style basis the new headings should have the first letter of each word capitalised as per the style guide for headings.
- The new headings do not respect the user "Settings mode" value, and based on hardware in use used can see headings with no options under those headings.
- The new headings reinforces that settings in menus should be reorganised into different Setup menus.  For example, all the settings relating to "Trash" and/or "Trashcan" should be moved to a new Setup screen that only deals with the trash settings.
- I don't think this change actually addresses the issue raised by Wedebe.  The Recordings and Zap settings are still intermixed under the same heading.  It is up to Wedebe to confirm if they are satisfied by this change.

I am not proposing that these changes be reverted but I am proposing that the changes should be fixed to follow the heading style and stop headings from appearing if there is no content under that heading.  I further propose that any settings that are better grouped under different Setup setting screens can have the new screens created and the entries left in both places until users get used to the new locations, at which point the old locations can be removed.

Thoughts?

**Comment by IanSav** [comment]
Most issues corrected in the latest update.   :)

I still think some more optimisations can be made but this can be handled as the specific modules get updated.
